ivre/db/elastic.py

```diff
@@ -94,7 +94,7 @@ def init(self):
                         "dynamic_templates": [
                             {"strings": {
                                 "match_mapping_type": "string",
-                                "mapping": {"type": "keyword"},
+                                "mapping": {"type": "keyword", "ignore_above": 32000},
```

Comment:
```suggestion
                                # prevent RequestError exceptions when
                                # one term's UTF-8 encoding is bigger
                                # than the max length 32766
                                "mapping": {"type": "keyword", "ignore_above": 32000},
```

Reply:
prevent RequestError exceptions when a large document is stored (max length 32766)

Reply:
Can you commit the change if you approve it?